components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/EntitlementPolicyAdminService.java

```diff
@@ -425,10 +414,12 @@ public String[] getAllPolicyIds(String searchString) throws EntitlementException
      */
     public PublisherDataHolder getSubscriber(String subscribeId) throws EntitlementException {
 
-        PolicyPublisher publisher = EntitlementAdminEngine.getInstance().getPolicyPublisher();
```

Comment:
Should handle similar to https://github.com/wso2/carbon-identity-framework/pull/5673/files#r1594986499